schulcloud/Controllers/ShortNotificationViewController.swift

```diff
@@ -97,6 +97,9 @@ extension ShortNotificationViewController : ViewControllerHeightDataSource {
     }
 }
 
+extension ShortNotificationViewController : PermissionInfoDataSource {
+    static var managedPermissions : UserPermissions { return UserPermissions.notificationView }
```

Comment:
This has to be `requiredPermissions`!

You don't have to use `var` to conform to the protocol. A shorter way to write this would be:
```swift
static let requiredPermissions = UserPermissions.notificationView
```